azurerm/resource_arm_iot_dps_cerificate.go

```diff
@@ -0,0 +1,149 @@
+package azurerm
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+
+	"github.com/Azure/azure-sdk-for-go/services/provisioningservices/mgmt/2018-01-22/iothub"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/validate"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmIotDPSCertificate() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmIotDPSCertificateCreateOrUpdate,
+		Read:   resourceArmIotDPSCertificateRead,
+		Update: resourceArmIotDPSCertificateCreateOrUpdate,
+		Delete: resourceArmIotDPSCertificateDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.IoTHubName,
+			},
+
+			"resource_group_name": azure.SchemaResourceGroupNameDiffSuppress(),
```

Comment:
is there a reason this is case insensitive?

```suggestion
			"resource_group_name": azure.SchemaResourceGroupName(),
```

Reply:
Nerp! Swapped